ompi/Makefile.am

```diff
@@ -128,9 +129,9 @@ DIST_SUBDIRS = \
 
 # Build the main MPI library
 
-lib_LTLIBRARIES = libmpi.la
-libmpi_la_SOURCES =
-libmpi_la_LIBADD = \
+lib_LTLIBRARIES = lib@OMPI_LIB_PREFIX@.la
```

Comment:
We were hoping to use this mechanism to rename `libmpi` to `libmpi_ibm` - so really a postfix. When preparing the patch I made it a bit more broad to allow both prefix and postfix by replacing the whole name. However, the variable `OMPI_LIB_PREFIX` is misleading so adopting `OMPI_LIBMPI_NAME` or `OMPI_LIBRARY_NAME` is probably better.

I can make that change once we settle on what the variable should be called.